opencti-platform/opencti-front/src/private/components/workspaces/WorkspaceShareList.tsx

```diff
@@ -0,0 +1,141 @@
+import { copyToClipboard } from '../../../utils/utils';
+import ItemMarkings from '../../../components/ItemMarkings';
+
+export const workspaceShareListQuery = graphql`
+  query WorkspaceShareListQuery($filters: FilterGroup) {
+    publicDashboards(filters: $filters) {
+      edges {
+        node {
+          id
+          uri_key
+          name
+          user_id
+          created_at
+          updated_at
+          allowed_markings {
+            id
+            definition
+            x_opencti_color
+          }
+        }
+      }
+    }
+  }
+`;
+
+interface WorkspaceShareListProps {
+  queryRef: PreloadedQuery<WorkspaceShareListQuery>
+  onDelete: (id: string) => void
+}
+
+const WorkspaceShareList = ({ queryRef, onDelete }: WorkspaceShareListProps) => {
+  const theme = useTheme<Theme>();
+  const { t_i18n, fld } = useFormatter();
+  const { publicDashboards } = usePreloadedQuery(workspaceShareListQuery, queryRef);
+  const dashboards = publicDashboards?.edges
+    .map((edge) => edge.node)
+    .sort((a, b) => a.created_at.localeCompare(b.created_at));
+
+  const copyLinkUrl = (uriKey: string) => {
+    copyToClipboard(
+      t_i18n,
+      `${window.location.origin}/public/dashboard/${uriKey}`,
+    );
+  };
+
+  if (!dashboards || dashboards.length === 0) {
+    return <p>{t_i18n('No public dashboard created yet')}</p>;
+  }
+
+  return (
+    <div style={{ marginTop: '12px', display: 'flex', gap: '12px', flexDirection: 'column' }}>
+      {dashboards.map((dashboard) => (
+        <Paper
+          key={dashboard.uri_key}
+          variant="outlined"
+          sx={{ padding: '12px' }}
+        >
+          <div style={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            alignItems: 'start',
+            marginBottom: '8px',
+          }}
+          >
+            <div>
+              <Typography variant="body1">
+                {dashboard.name}
+              </Typography>
+              <Typography variant="body2">
+                public/dashboard/{dashboard.uri_key}
+              </Typography>
+            </div>
+
+            <ToggleButtonGroup size="small">
+              <Tooltip title={t_i18n('Copy link')}>
+                <ToggleButton
+                  aria-label="Label"
+                  size="small"
+                  value="copy-link"
+                  onClick={() => copyLinkUrl(dashboard.uri_key)}
+                >
+                  <ContentCopyIcon fontSize="small" color="primary" />
+                </ToggleButton>
+              </Tooltip>
+              {/* <Tooltip title={t_i18n('Disable public dashboard')}> */}
+              {/*  <ToggleButton */}
+              {/*    aria-label="Label" */}
+              {/*    size="small" */}
+              {/*    value="disable-link" */}
+              {/*  > */}
+              {/*    <DoNotDisturbAltIcon fontSize="small" color="primary" /> */}
+              {/*  </ToggleButton> */}
+              {/* </Tooltip> */}
```

Comment:
Do you keep that in purpose for next chuncks ?

Reply:
Yep